explorer/websockethandlers.go

```diff
@@ -59,55 +60,107 @@ func (exp *explorerUI) RootWebsocket(w http.ResponseWriter, r *http.Request) {
 
 				// handle received message according to event ID
 				var webData WebSocketMessage
+				//  If the request sent is past the limit exit the loop
```

Comment:
Not right to say "exit" the loop since you are using `continue`.  This goes to the next iteration, or back to the beginning of the loop"

Reply:
ok. Noted